client/hwlib/debian/tests/run-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Nitpick, this script looks like something that would work just fine with `/bin/sh`